src/Microsoft.VisualStudio.Services.Agent/AgentService.cs

```diff
@@ -19,8 +19,8 @@ public interface IAgentService
 
     public abstract class AgentService
     {
-        protected IHostContext HostContext { get; private set; }
```

Comment:
Alternatively, the SecretMasker doesn't need to store the HostContext since it ~~doesn't need it for anything~~ shouldn't call it for anything.